src/ResourceManagement/SqlManagement/Microsoft.Azure.Management.Sql/Generated/Models/ElasticPool.cs

```diff
@@ -0,0 +1,117 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for
+// license information.
+// 
+// Code generated by Microsoft (R) AutoRest Code Generator 0.17.0.0
+// Changes may cause incorrect behavior and will be lost if the code is
+// regenerated.
+
+namespace Microsoft.Azure.Management.Sql.Models
+{
+    using System.Linq;
+
+    /// <summary>
+    /// Represents an Azure SQL Database elastic pool.
+    /// </summary>
+    [Microsoft.Rest.Serialization.JsonTransformation]
+    public partial class ElasticPool : Resource
+    {
+        /// <summary>
+        /// Initializes a new instance of the ElasticPool class.
+        /// </summary>
+        public ElasticPool() { }
+
+        /// <summary>
+        /// Initializes a new instance of the ElasticPool class.
+        /// </summary>
+        /// <param name="location">Resource location</param>
+        /// <param name="name">Resource name</param>
+        /// <param name="id">Resource Id</param>
+        /// <param name="type">Resource type</param>
+        /// <param name="tags">Resource tags</param>
+        /// <param name="creationDate">The creation date of the Azure SQL
+        /// Elastic Pool (ISO8601 format).</param>
+        /// <param name="state">The state of the Azure SQL Elastic Pool.
+        /// Possible values include: 'Creating', 'Ready', 'Disabled'</param>
+        /// <param name="edition">The edition of the Azure SQL Elastic Pool.
+        /// Possible values include: 'Basic', 'Standard', 'Premium'</param>
```

Comment:
This is generated from the swagger, so I will address this in a separate PR for the specs.